scripts/encoding/encoder.py

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+import pandas as pd
+from sklearn.feature_extraction.text import TfidfVectorizer, CountVectorizer
+
+ENCODER_TOOLS = {
```

Comment:
Les autres modules auront probablement pas besoin d'utiliser cette constante (privée au package), on peut ajouter un underscore pour le montrer. (Penser à modifier le reste du fichier)
```suggestion
_ENCODER_TOOLS = {
```